Ghidra/RuntimeScripts/build.gradle

```diff
@@ -42,6 +42,12 @@ rootProject.PLATFORMS.each { platform ->
 			t.from (p.file("Linux/ghidraRun")) 
 		}
 
+		if (isMac(platform.name)) {
```

Comment:
I've added a custom macOS directory so this will only be bundled if you build for macOS. I've also prefixed the script name with `macos_` to make this clear. Again, happy to rename the script if you have a suggestion.